diff --git a/Dockerfile b/Dockerfile
index 1265220..dacf124 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -2,10 +2,10 @@ FROM oven/bun:1.2.19-alpine AS builder
 WORKDIR /app
 
 COPY ./package.json ./bun.lock ./
-RUN bun install --frozen-lockfile
+RUN bun install --frozen-lockfile --ignore-scripts
 
 COPY . .
-RUN bun run build
+RUN bun ./node_modules/tsdown/dist/run.mjs
 
 FROM oven/bun:1.2.19-alpine AS runner
 WORKDIR /app
